README.md

```diff
@@ -1,57 +1,71 @@
-# Project Name
-
-(short, 1-3 sentenced, description of the project)
-
-## Features
-
-This project framework provides the following features:
-
-* Feature 1
-* Feature 2
-* ...
+---
+name: "Azure Functions JavaScript HTTP Trigger using AZD"
+description: This repository contains a Azure Functions HTTP trigger quickstart written in JavaScript and deployed to Azure using Azure Developer CLI (AZD). This sample uses manaaged identity and a virtual network to insure it is secure by default.
+page_type: sample
+languages:
+- nodejs
+- JavaScript
+products:
+- azure
+- azure-functions
+- entra-id
+urlFragment: functions-quickstart-javascript-azd
+---
+
+# Azure Functions JavaScript HTTP Trigger using AZD
+
+This repository contains a Azure Functions HTTP trigger quickstart written in JavaScript and deployed to Azure using Azure Developer CLI (AZD). This sample uses manaaged identity and a virtual network to insure it is secure by default. 
 
 ## Getting Started
 
 ### Prerequisites
 
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
-
-### Installation
-
-(ideally very short)
-
-- npm install [package name]
-- mvn install
-- ...
-
-### Quickstart
-(Add steps to get up and running quickly)
-
-1. git clone [repository clone url]
-2. cd [repository name]
-3. ...
-
-
-## Demo
-
-A demo app is included to show how to use the project.
-
-To run the demo, follow these steps:
-
-(Add steps to start up the demo)
-
-1.
-2.
-3.
-
-## Resources
-
-(Any additional resources or related projects)
-
-- Link to supporting information
-- Link to similar sample
-- ...
+1) [Node.js 20](https://www.nodejs.org/) 
+2) [Azure Functions Core Tools](https://learn.microsoft.com/azure/azure-functions/functions-run-local?tabs=v4%2Cmacos%2Ccsharp%2Cportal%2Cbash#install-the-azure-functions-core-tools)
+3) [Azure Developer CLI (AZD)](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd)
+4) [Visual Studio Code](https://code.visualstudio.com/) - Only required if using VS Code to run locally
+
+## Run on your local environment
+
+### Prepare your local environment
+1) Add a file named local.settings.json file to the root of the project with the following contents. This will allow you to run your function locally.
+```json
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "node"
+  }
+}
+```
+
+### Using Functions CLI
+1) Open this folder in a new terminal and run the following commands:
```

Comment:
I assume for this readme, the user will just clone and be at step 1 where they can func start to run, right?  

As we create the doc for this, there will be one more step which is to `azd init --template functions-quickstart-javascript-azd` right?  It would be handy to test that now just to make sure nothing is busted.  But we could always PR changes for that after this.

Reply:
Yes to clone for now. In order to azd init, I need to do the publishing work to awesome-azd and then I will update this repo with instructions to use azd init.